SilKit/source/services/orchestration/TimeSyncService.cpp

```diff
@@ -546,6 +612,75 @@ bool TimeSyncService::AbortHopOnForCoordinatedParticipants() const
     }
     return false;
 }
+
+auto TimeSyncService::GetCurrentWallClockSyncPoint() const -> std::chrono::nanoseconds
+{
+    Lock lock{_mx};
+    return _currentWallClockSyncPoint;
+}
+
+void TimeSyncService::StartWallClockCouplingThread()
+{
+    _wallClockCouplingThreadRunning = true;
+    _stopFuture = _wallClockCouplingThreadStopPromise.get_future();
+
+    _wallClockCouplingThread = std::thread{[&]() {
```

Comment:
The catch-all captures, particularly by reference, are in almost all cases a bad idea. They, e.g., make it super-easy to accidentially use stack-variables after the caller returned.